app/media/[slug]/page.tsx

```
@@ -0,0 +1,73 @@
+  if (!prop) {
+    console.warn("No prop provided");
+    return;
+  }
+
+  // Initialize urlString
+  let urlString = "";
+
+  // Determine the URL based on the type and value of prop
+  if (typeof prop === "string") {
+    // Check if it's a valid number string
+    if (!isNaN(Number(prop))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)